### PR TITLE
feat(owner): 🎸 Allow only rent, charges and living space > 0

### PR DESCRIPTION
**Pull request by mattboll**

### CODE REVIEW

File: owner/src/components/property/PropertyLivingSpace.vue [hunk 1]
```diff
@@ -77,13 +78,15 @@ function onBack() {
...
+    "living-space-placeholder": "size m²",
+    "number-not-positive": "Living space must be more than zero"
   },
   "fr": {
     "living-space-title": "Surface",
     "living-space-subtitle": "Quelle est la surface, loi Carrez",
     "living-space-label": "Surface de ma propriété, loi Carrez",
-    "living-space-placeholder": "taille en m²"
+    "living-space-placeholder": "taille en m²",
+    "number-not-positive": "La surface doit supérieur à zéro"
```
Comment (mat007) [inline owner/src/components/property/PropertyLivingSpace.vue:89]: Maybe
```
     "number-not-positive": "La surface doit être supérieure à zéro"
```
?

File: owner/src/components/property/ValidateProperty.vue [hunk 2]
```diff
@@ -144,6 +163,7 @@ function onBack() {
     "property-furniture": "Ameublement",
     "property-address": "Adresse du logement",
     "property-living-space": "Surface",
+    "property-rent": "Loyer et charges mensuels",
     "try-again": "Veuillez fixer les erreurs pour soumettre la propriété"
```
Comment (mat007) [inline owner/src/components/property/ValidateProperty.vue:167]:
```
     "try-again": "Veuillez corriger les erreurs pour soumettre la propriété"
```

Comment (mattboll) [reply owner/src/components/property/ValidateProperty.vue:167]: :+1:

File: owner/src/components/property/PropertyLivingSpace.vue (hunk 1 above)
Comment (mattboll) [reply owner/src/components/property/PropertyLivingSpace.vue:89]: Yes, obviously, thank you :)

**Pull request merged**